### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by tysonkamp**
Converted to Python.

### CODE REVIEW

File: generateParantheses/Python/generateBalancedParenthesis.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# "(()())",
+# "(())()",
+# "()(())",
+# "()()()"
+# ]
+
+#/usr/bin/python
+
+def generateParenthesis(n):
+	"Bootstrap the recursive process that will generate permutations of string with "n" balanced parenthesis"
```
Comment (mohitathwani) [inline generateParantheses/Python/generateBalancedParenthesis.py:19]: I'm not a Python expert, could you please tell me what this string is? Is it a comment?

File: generateParantheses/Python/generateBalancedParenthesis.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	result = []
+	s = ""
+
+	backtrack(n, s, result)
+	return result
+
+def backtrack(n, s, result, numOpen = 0, numClose = 0):
+	"Add comment"
```
Comment (mohitathwani) [inline generateParantheses/Python/generateBalancedParenthesis.py]: Same here

File: generateParantheses/Python/generateBalancedParenthesis.py (hunk 1 above)
Comment (tysonkamp) [reply generateParantheses/Python/generateBalancedParenthesis.py:19]: It's just a descriptive line to explain what the function does.  "The first statement of the function body can optionally be a string literal; this string literal is the function’s documentation string, or docstring. " , from this page: 
 https://docs.python.org/3/tutorial/controlflow.html#defining-functions

File: generateParantheses/Python/generateBalancedParenthesis.py (hunk 2 above)
Comment (tysonkamp) [reply generateParantheses/Python/generateBalancedParenthesis.py]: I forgot to add the actual comment!  This is supposed to document what the function does.

**Comment by tysonkamp** [comment]
I just added a comment for the "backtrack" function.

**Pull request merged**